cmd-info.c

```diff
@@ -802,7 +802,7 @@ static int read_pattern_type(void *arg)
 {
 	struct ftrace_file_handle *handle = arg;
 	struct uftrace_info *info = &handle->info;
-	char buf[4096];
+	char buf[BUF_SIZE];
```

Comment:
I think it's better to pass a buffer instead of having a local buffer for each function.

Reply:
Oh, so do I !